core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -246,8 +246,10 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
   }
 
   private def logAuditMessage(principal: KafkaPrincipal, authorized: Boolean, operation: Operation, resource: Resource, host: String) {
-    val permissionType = if (authorized) "Allowed" else "Denied"
-    authorizerLogger.debug(s"Principal = $principal is $permissionType Operation = $operation from host = $host on resource = $resource")
+    if (authorized && authorizerLogger.isDebugEnabled)
+      authorizerLogger.debug(s"Principal = $principal is Allowed Operation = $operation from host = $host on resource = $resource")
+    else if (!authorized && authorizerLogger.isInfoEnabled)
+      authorizerLogger.info(s"Principal = $principal is Denied Operation = $operation from host = $host on resource = $resource")
```

Comment:
You can write it like:

```
  def logMessage: String = {
    val result = if (authorized) "Allowed" else "Denied"
    s"Principal = $principal is $result Operation = $operation from host = $host on resource = $resource"
  }

  if (authorized) authorizerLog.debug(logMessage)
  else authorizerLog.info(logMessage)
```

Reply:
Thanks for the suggestion.  this is much cleaner now. Updated the PR.